File: aroundHubSpringBoot/src/main/java/com/example/aroungHubJavaSpringBoot/aroundHubSpringBoot/service/ProductService.java
package com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.service;

import com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.dto.ProductDto;

public interface ProductService {

    ProductDto saveProduct(String productId, String productName, int productPrice, int productStock);

    ProductDto getProduct(String productId);
}

File: aroundHubKotlinSpringBoot/src/main/kotlin/com/example/aroungHubKotlinSpringBoot/aroundHubKotlinSpringBoot/service/ProductService.kt
package com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.service

import com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.dto.ProductDto

interface ProductService {
    fun saveProduct(productId: String, productName : String, productPrice: Int, productStock: Int) : ProductDto

    fun getProduct(productId: String) : ProductDto
}